website/content/docs/connect/config-entries/jwt-provider.mdx

```diff
@@ -436,6 +511,79 @@ Specifies a jittered exponential backoff strategy. When this field is empty, Env
 | `BaseInterval`| Specifies the base interval to use for the next back off computation. | String | `1s` |
 | `MaxInterval` | Specifies the maximum interval between retries. By default, this value is 10 times `BaseInterval`. | String | `10s` |
 
+### `JSONWebKeySet{}.Remote{}.JWKSCluster`
+
+Defines how the specified Remote JWKS URI is to be fetched.
+
+#### Values
+
+- Default: None
+- Data type: Map that can contain the following parameters:
```

Comment:
nit: Not sure if the other docs use the same language as here but it may be more accurate to call these `keys` instead of `parameters`

Reply:
I think it is probably just a language convention that I am used to. A map to me is a collection of key/value pairs as in the specific `map` datatype. 

I have never heard them called parameters but if that is the general term we have standardized around then it is good to me.